cmd/crdvalidator/annotation/annotation.go

```diff
@@ -0,0 +1,23 @@
+package annotation
+
+import (
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	ValidationKey = "core.rukpak.io/enable-safe-upgrade-validation"
```

Comment:
nit: now that the flag is `enabled` I don't think we should have `enable` in the key as well. I also think we should include the word `crd` in here just so it's more clear what is being validated. Also, in the control plane call the preferred term was "update" instead of "upgrade" since there is no upgrade verb and it's more clear. So I think we can tweak this a bit more.

How does something like `"core.rukpak.io/safe-crd-updates"` sound?

Reply:
I agree so much that I did this on accident before reading this comment 😆 Just pushed it in the most recent change 👍🏻